articles/universal-login/new-experience-limitations.md

```diff
@@ -0,0 +1,25 @@
+---
+description: New Universal Login Limitations
+topics:
+  - login
+  - universal-login
+contentType: index
+toc: true
+---
+# New Universal Login Experience Limitations
```

Comment:
We don't support MFA Enrollment tickets/invitations. The MFA enrollment will be performed using the classic experience when using enrollment tickets / invitations.